R/epipredict-data.R

```diff
@@ -51,31 +55,72 @@
 #'   \item{abbr}{Postal abbreviation for the location}
 #' }
 #'
-#' @source United States Census Bureau, at
+#' @source United States Census Bureau, Population Division, at
 #'   \url{https://www2.census.gov/programs-surveys/popest/datasets/2010-2019/counties/totals/co-est2019-alldata.pdf},
 #'   \url{https://www.census.gov/data/tables/time-series/demo/popest/2010s-total-puerto-rico-municipios.html},
 #'   and \url{https://www.census.gov/data/tables/2010/dec/2010-island-areas.html}
+#'   This data is public domain.
 "state_census"
 
 # Epipredict Vignette Data ----------------------------------------------------
 
-#' CTIS COVID Behaviours
+#' Subset of CTIS COVID-19 behaviours from 5 states
 #'
 #' Data set for a handful of states on masking and distancing behaviours
-#' during the COVID-19 Pandemic and downloaded from the CMU Delphi COVIDcast
-#' Epidata API. This data set covers the period from
-#' June to December 2021.
+#' during the COVID-19 Pandemic, and downloaded from the CMU Delphi COVIDcast
+#' Epidata API. This data set covers the period from June to December 2021,
+#' and is limited to California, Florida, Texas, New Jersey, and New York.
+#'
+#' @format An object of class \code{tbl_df} (inherits from \code{tbl}, \code{data.frame}) with 1055 rows and 4 columns.
+#' @section Data dictionary:
+#' A tibble with columns:
+#' \describe{
+#'   \item{geo_value}{the geographic value associated with each row
+#'       of measurements.}
+#'   \item{time_value}{the time value associated with each row of measurements.}
+#'   \item{masking}{Estimated percentage of people who wore a mask for most or all of the time while in public in the past 7 days; those not in public in the past 7 days are not counted.}
+#'   \item{distancting}{Estimated percentage of respondents who reported that all or most people they enountered in public in the past 7 days maintained a distance of at least 6 feet. Respondents who said that they have not been in public for the past 7 days are excluded.}
+#' }
```

Comment:
note: Moved the data dictionary part of the `@format` to a different section to make doc inheritance work correctly in `epiprocess` and `epipredict`. See https://stackoverflow.com/questions/76468979/how-to-inherit-roxygen-documentation-from-data/76469156#76469156